realm/realm-library/src/main/java/io/realm/BaseRealm.java

```diff
@@ -226,9 +228,8 @@ protected void removeHandler() {
      * the last transaction was committed.
      *
      * @param destination file to save the Realm to.
-     * @throws java.io.IOException if any write operation fails.
      */
-    public void writeCopyTo(File destination) throws java.io.IOException {
+    public void writeCopyTo(File destination) {
```

Comment:
Your right, that doesn't make sense. I think the main reason I was concerned about IOEception for `writeCopyTo` is that the the `File` param could cause it, not opening the Realm file itself